include/grpc++/impl/codegen/config.h

```diff
@@ -54,6 +54,7 @@
 // nullptr was added in gcc 4.6
 #if (__GNUC__ * 100 + __GNUC_MINOR__ < 406)
 #define GRPC_CXX0X_NO_NULLPTR 1
+#define GRPC_CXX0X_LIMITED_TOSTRING 1
```

Comment:
gcc4.4 only supports std::to_string(long long), (long long unsigned), or (long double). So, for int and unsigned int types, there was an ambiguous conversion.